Disable argument-mismatch errors for SPEC CPU2006 416.gamess for GCC >= 8

gfortran 8 is able to detect a new warning/error that is causing SPEC CPU2006 416.gamess to fail:

```
ecp.fppized.f:402:15:

       CALL ZFN(ZFNLM,NPNP-1,ZLM,LMF,LMX,LMY,LMZ)
               1
Error: Actual argument contains too few elements for dummy argument 'zfnlm' (121/125) at (1)
ecp.fppized.f:544:18:

          CALL ZFN(ZFNLM,NPNP-1,ZLM,LMF,LMX,LMY,LMZ)
                  1
Error: Actual argument contains too few elements for dummy argument 'zfnlm' (121/125) at (1)
ecp.fppized.f:739:24:

                CALL ZFN(ZFNLM,NPNP-1,ZLM,LMF,LMX,LMY,LMZ)
                        1
Error: Actual argument contains too few elements for dummy argument 'zfnlm' (121/125) at (1)
ecp.fppized.f:994:18:

          CALL ZFN(ZFNLMC,NPCPL-1,ZLM,LMF,LMX,LMY,LMZ)
                  1
Error: Actual argument contains too few elements for dummy argument 'zfnlm' (121/125) at (1)
ecp.fppized.f:998:18:

          CALL ZFN(ZFNLMB,NPBPL-1,ZLM,LMF,LMX,LMY,LMZ)
                  1
Error: Actual argument contains too few elements for dummy argument 'zfnlm' (121/125) at (1)
specmake: *** [ecp.fppized.o] Error 1
specmake: *** Waiting for unfinished jobs....
```

As far as I can see, we need  to pass -Wno-error=argument-mismatch to 416.gamess.

--- a/fvtr/speccpu2006/at10.0-sniff32.cfg
+++ b/fvtr/speccpu2006/at10.0-sniff32.cfg
@@ -51,6 +51,9 @@ strict_rundir_verify=0
 CPORTABILITY   = -DSPEC_CPU_LINUX_PPC -std=gnu89
 EXTRA_CFLAGS = -fno-strict-aliasing
 
+416.gamess=default=default=default:
+FPORTABILITY = -std=legacy
+
 462.libquantum=default=default=default:
 CPORTABILITY   = -DSPEC_CPU_LINUX
 

--- a/fvtr/speccpu2006/at10.0-sniff64.cfg
+++ b/fvtr/speccpu2006/at10.0-sniff64.cfg
@@ -57,6 +57,9 @@ EXTRA_CFLAGS = -fno-strict-aliasing
 403.gcc=default=default=default:
 PORTABILITY = -DSPEC_CPU_LP64
 
+416.gamess=default=default=default:
+FPORTABILITY = -std=legacy
+
 462.libquantum=default=default=default:
 CPORTABILITY   = -DSPEC_CPU_LINUX
 

